core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,16 +18,40 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
```

Comment:
core/templates/dev/head/pages/exploration_editor/exploration_editor.html that also uses inline css. Some elements are in page/base.html, so including this css in oppia.css will effect whole body. As it was mentioned by @seanlip .